apps/api/src/persistence/repositories/arango-repository.provider.ts

```diff
@@ -114,9 +113,14 @@ export class ArangoRepositoryProvider implements IRepositoryProvider {
             AggregateType.song,
             AggregateType.digitalText,
             AggregateType.term,
+            AggregateType.vocabularyList,
         ];
 
         if (eventSourcedAggregateTypes.includes(resourceType)) {
+            /**
+             * Let's review the relationship of event and snapshot
```

Comment:
See other comment.

Reply:
Linked to [this story](https://github.com/COSCRAD/coscrad/pull/638#discussion_r1962056329).